lib/src/browser/disposable_browser.dart

```diff
@@ -184,6 +184,11 @@ class Disposable implements disposable_common.Disposable {
       _disposable.getManagedTimer(duration, callback);
 
   @override
+  StreamSubscription<T> getManagedStreamSubscription<T>(
+          Stream<T> stream, void onData(T event)) =>
```

Comment:
🔥 You need optional params for `onError`, `onDone`, etc... 

I wonder if the Dart SDK has a typedef for the listen method on streams.